project/Dependencies.scala

```diff
@@ -5,7 +5,7 @@ import sbt._
 object Dependencies {
   val identityLibVersion = "3.254"
   val awsVersion = "1.12.205"
-  val capiVersion = "19.0.0"
+  val capiVersion = "19.0.3"
   val faciaVersion = "4.0.0"
```

Comment:
```suggestion
  val faciaVersion = "4.0.1"
```

Probably not that important - but I did https://github.com/guardian/facia-scala-client/pull/276 to make sure a small class-hierarchy change in the CAPI client didn't cause any problems!